src/studio/src/designer/frontend/dashboard/features/createService/CreateService.tsx

```diff
@@ -1,17 +1,15 @@
-import { makeStyles } from '@material-ui/core/styles';
-import { Grid } from '@material-ui/core';
 import React from 'react';
-
+import { makeStyles } from '@mui/styles';
+import { Grid } from '@mui/material';
 import { useAppSelector } from 'common/hooks';
-
 import AltinnButton from 'app-shared/components/AltinnButton';
 import AltinnSpinner from 'app-shared/components/AltinnSpinner';
 import { getLanguageFromKey } from 'app-shared/utils/language';
 
 import { ServiceOwnerSelector } from './ServiceOwnerSelector';
 import { RepoNameInput } from './RepoNameInput';
 // import { RepoTypeSelector } from './RepoTypeSelector';
```

Comment:
```suggestion
```

Reply:
Dunno what this was supposed to be... but I am deleting it. The file can be found in git history anyway.